site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -250,6 +250,8 @@ When a device has successfully opened a receiver link for commands, the adapter
 
 Devices send their responses to commands by means of sending an AMQP message with properties specific to the command that has been executed. The AMQP adapter accepts responses being published using either *at most once* (QoS 0) or *at least once* (QoS 1) delivery semantics. The device must send the command response messages using the same (sender) link that it uses for sending telemetry data and events.
 
+The AMQP adapter checks the configured [message limit] ({{< relref "/concepts/resource-limits.md" >}}) before accepting any command requests and responses. In case of incoming command requests from business applications or the command responses from devices, if the message limit is exceeded, the Adapter rejects the message with the reason `amqp:resource-limit-exceeded`. 
```

Comment:
didn't we already describe this in the *Message Limits* section at the top of the page?

Reply:
_Message Limits_ in the top section gives only a short description of message limits. In addition to that, each section corresponding to `telemetry` and `event` contain information regarding the outcome of the message limit exceeded e.g _the message is rejected and the reason is_ `amqp:resource-limit-exceeded`. In this PR, it has been updated for the _commands_ section.

Reply:
I see :+1: